src/itest/kotlin/nl/info/zac/itest/NotificationsZaakDestroyTest.kt

```diff
@@ -0,0 +1,141 @@
+import nl.info.zac.itest.config.ItestConfiguration.ZAC_API_URI
+import okhttp3.Headers
+import org.json.JSONObject
+import java.time.ZoneId
+import java.time.ZonedDateTime
+import java.util.UUID
+import kotlin.time.Duration.Companion.seconds
+
+/**
+ * This test creates a zaak and because we do not want this test to impact [ZoekenRESTServiceTest]
+ * we run it afterward.
+ */
+@Order(TEST_SPEC_ORDER_AFTER_SEARCH)
+class NotificationsZaakDestroyTest : BehaviorSpec({
+    val logger = KotlinLogging.logger {}
+    val itestHttpClient = ItestHttpClient()
+    val zacClient = ZacClient()
+
+    Given("Existing process data and Solr index data for a specific zaak in ZAC") {
+        lateinit var zaakUUID: UUID
+        lateinit var zaakIdentificatie: String
+        zacClient.createZaak(
+            zaakTypeUUID = ZAAKTYPE_INDIENEN_AANSPRAKELIJKSTELLING_DOOR_DERDEN_BEHANDELEN_UUID,
+            groupId = TEST_GROUP_A_ID,
+            groupName = TEST_GROUP_A_DESCRIPTION,
+            startDate = DATE_TIME_2024_01_31
+        ).run {
+            val responseBody = body!!.string()
+            logger.info { "Response: $responseBody" }
+            this.isSuccessful shouldBe true
+            JSONObject(responseBody).run {
+                zaakUUID = getString("uuid").run(UUID::fromString)
+                zaakIdentificatie = getString("identificatie")
+            }
+        }
+        // reindex so that the new zaak gets added to the Solr index
+        itestHttpClient.performGetRequest(
+            url = "$ZAC_API_URI/indexeren/herindexeren/ZAAK",
+            addAuthorizationHeader = false
+        ).run {
+            logger.info { "Response: ${body!!.string()}" }
+            this.isSuccessful shouldBe true
+        }
+        // wait for the indexing to complete
+        eventually(10.seconds) {
+            val searchResponseBody = itestHttpClient.performPutRequest(
+                url = "$ZAC_API_URI/zoeken/list",
+                requestBodyAsString = """
+                    {
+                        "filtersType": "ZoekParameters",
+                        "alleenMijnZaken": false,
+                        "alleenOpenstaandeZaken": true,
+                        "alleenAfgeslotenZaken": false,
+                        "alleenMijnTaken": false,
+                        "zoeken": { "ALLE": "$zaakIdentificatie" }, 
+                        "filters": {},                            
+                        "datums": {},
+                        "rows": 10,
+                        "page": 0                        
+                    }
+                """.trimIndent()
+            ).body!!.string()
+            JSONObject(searchResponseBody).getInt("totaal") shouldBe 1
+            with(JSONObject(searchResponseBody).getJSONArray("resultaten").getJSONObject(0).toString()) {
+                shouldContainJsonKeyValue("identificatie", zaakIdentificatie)
+            }
```

Comment:
Can this be done with JSONPath?

I imagine something like this can work:
```suggestion
            searchResponseBody.shouldContainJsonKeyValue("$.resultaten[0].identificatie", zaakIdentificatie)
```